pkg/controller/nodeipam/ipam/multi_subnetworks_handler.go

```diff
@@ -0,0 +1,131 @@
+package ipam
+
+import (
+	"context"
+	"fmt"
+	nodetopologyv1 "github.com/GoogleCloudPlatform/gke-networking-api/apis/nodetopology/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilnode "k8s.io/cloud-provider-gcp/pkg/util/node"
+	"k8s.io/klog/v2"
+	"strings"
+)
+
+const nodeTopologyCRName = "default"
+
+func (ca *cloudCIDRAllocator) updateNodeTopology(node *v1.Node) error {
+	ctx := context.Background()
+
+	hasSubnetLabel, nodeSubnet := getNodeSubnetLabel(node)
```

Comment:
what is the purpose of checking here then again in L60? It's quite hard to read. Can we move L19-21 to after `if crSubnets==nil` block?

Reply:
Ack. We don't need the log here and I have handled it in the next check.